src/porepy/fracs/simplex.py

```diff
@@ -389,6 +393,8 @@ def triangle_grid(fracs, domain, do_snap_to_grid=False, **kwargs):
     # We find the end points that are shared by more than one intersection
     intersections = __find_intersection_points(lines_split)
 
+    intersections = np.empty(0)
```

Comment:
You overwrite the old line? Why? If correct, it is better to replace the first definition of intersections.

Reply:
this is definitly wrong, I cherry pick this commit and after there is another one to fix it. I cherry pick it as well